designer/server/src/main/scala/pl/touk/nussknacker/ui/config/scenariotoolbar/ScenarioToolbarsConfig.scala

```diff
@@ -78,6 +79,8 @@ object ToolbarConditionType extends Enumeration {
 final case class ToolbarCondition(
     fragment: Option[Boolean],
     archived: Option[Boolean],
+    always: Option[Boolean],
```

Comment:
What does it mean? I thought that lack of conditions means that it is always visible / enabled